thrift/src/main/java/com/linecorp/armeria/server/thrift/THttpServiceBuilder.java

```diff
@@ -115,8 +118,23 @@ public THttpServiceBuilder addService(Object implementation) {
      */
     public THttpServiceBuilder otherSerializationFormats(SerializationFormat... otherSerializationFormats) {
         requireNonNull(otherSerializationFormats, "otherSerializationFormats");
+        otherSerializationFormats(ImmutableList.copyOf(otherSerializationFormats));
+        return this;
+    }
+
+    /**
+     * Adds other {@link SerializationFormat} to the builder. Current supported {@link SerializationFormat}s are
+     * {@link ThriftSerializationFormats#values()}. If nothing is specified then all the
+     * {@link SerializationFormat#values()}s are added.
+     *
+     * <p>Currently, the only way to specify a serialization format is by using the HTTP session
+     * protocol and setting the Content-Type header to the appropriate {@link SerializationFormat#mediaType()}.
+     */
+    public THttpServiceBuilder otherSerializationFormats(Iterable<SerializationFormat>
+                                                                 otherSerializationFormats) {
+        requireNonNull(otherSerializationFormats, "otherSerializationFormats");
         this.otherSerializationFormats = new LinkedHashSet<>();
```

Comment:
This line needs to be removed, because otherwise a user can't do the following:

```java
builder.otherSerializationFormats(ImmutableSet.of(fmt1, fmt2));
builder.otherSerializationFormats(ImmutableSet.of(fmt3, fmt3));
```